### PR TITLE
TMA-565: Provisioning brick doesn't add technical users when input source contains project_id

### PR DESCRIPTION
**Pull request by vhtien**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/94/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/942/ : SUCCESS in 20s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/88/ : SUCCESS in 16s

File: spec/unit/actions/ensure_technical_users_project_spec.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      result = subject.class.call(params)
+      expected = [{ project: 'foo project',
+                    pid: 'abcdefg',
+                    login: 'foo@bar.com',
+                    role: 'admin',
+                    result: 'successful',
+                    message: 'yahoo!',
+                    url: '/gdc/account/foo' }]
+      expect(result).to eq(expected)
+    end
```
Comment (panjan) [inline spec/unit/actions/ensure_technical_users_project_spec.rb]: We should also check that the user is created in that project. Something like this: `expect(project).to receive(:create_users).with({ login: 'foo@bar.com' })`

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/101/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/954/ : SUCCESS in 17s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/100/ : SUCCESS in 13s

File: spec/unit/actions/ensure_technical_users_project_spec.rb (hunk 1 above)
Comment (vhtien) [inline spec/unit/actions/ensure_technical_users_project_spec.rb]: done

**Comment by panjan** [comment]
ok to merge

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/54/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/977/ : SUCCESS in 16s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/123/ : SUCCESS in 21s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/871/ : SUCCESS in 45m 34s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/645/ : UNSTABLE in 5m 05s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/122/ : SUCCESS in 5s
- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/979/ : SUCCESS in 17s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/125/ : SUCCESS in 20s

**Comment by vhtien** [comment]
extended test

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/123/ : SUCCESS in 5s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/872/ : SUCCESS in 45m 22s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/646/ : SUCCESS in 20m 33s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/56/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/981/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/127/ : SUCCESS in 20s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/874/ : SUCCESS in 44m 42s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/648/ : SUCCESS in 19m 52s

**Pull request merged**